backend/src/routes/index.js

```diff
@@ -62,8 +73,8 @@ router.get('/events/:eventId', async (req, res, next) => {
   let event = await Event.findOne({ _id: req.params.eventId })
 
   if (!event) return next(new Error('Event not found'))
-
-  event = Event.decorateForUser(event, req.session.user && req.session.computerId)
+  const singularities =  await checkSingularity({ userId: req.session.user._id, computerId: req.session.computerId, sessionId: req.session._id })
```

Comment:
singularities would be an array so the plural usage is correct. But what this returns is not singularities but rather singularity ids and user ids. So a better name to reflect this situation would be `userIds` as the variable name.